src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java

```diff
@@ -74,7 +74,8 @@ private void updateInsertedRowsColumn(int insertedRows, RowMap startBootstrapRow
 		long now = System.currentTimeMillis();
 		if ( now - lastInsertedRowsUpdateTimeMillis > INSERTED_ROWS_UPDATE_PERIOD_MILLIS ) {
 			long rowId = ( long ) startBootstrapRow.getData("id");
-			String sql = "update maxwell.bootstrap set inserted_rows = ?, binlog_file = ?, binlog_position = ? where id = ?";
+			String sql = "update `bootstrap` set inserted_rows = ?, binlog_file = ?, binlog_position = ? where id = ?";
+			connection.setCatalog(this.context.getConfig().databaseName);
```

Comment:
hmmm, we really shouldn't have to profilerate `setCatalog` calls out everywhere -- better to just to do a `setCatalog` on the connection once, after we check it out of the pool... unless there's something here I'm missing?